### PR TITLE
Add header for mockup

### PR DESCRIPTION
**Pull request by martypowell**
Sorry this is a big one

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,170 +1,226 @@
...
+      .pipe(uglify())
+      .on("error", err => {
+        util.log(util.colors.red("[Error]"), err.toString());
+      })
+      .pipe(rename({ suffix: ".min" }))
+      .pipe(gulp.dest(`${DIST_FOLDER}/js`))
+);
+
+gulp.task("process-app-js", () => {
+  // const appRootFolder = 'js/apps';
```
Comment (mxsnyder) [inline gulpfile.js]: Is this all commented out, but left in on purpose?

File: js/mobile-nav.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+namespacer("restaurantWeek");
```
Comment (mxsnyder) [inline js/mobile-nav.js]: Double quotes? ***faints***

File: js/mobile-nav.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      self.options.navigationListSelector || ".navigation-list";
+
+    $(document).on(
+      "click",
+      self.options.mobileNavButtonSelector,
+      onMobileNavClick
+    );
+  };
+
+  const onMobileNavClick = e => {
```
Comment (mxsnyder) [inline js/mobile-nav.js]: Please use a better name for the event than "e". For example, "clickEvent".

File: js/mobile-nav.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    /* test-code */
+    onMobileNavClick,
+    /* end-test-code */
+    init
+  };
+})(jQuery);
+
+//On Document Ready
+$(document).ready(() => {
+	restaurantWeek.mobileNav.init();
```
Comment (mxsnyder) [inline js/mobile-nav.js]: Nitpicking, but would you fix the spacing here?

File: stylesheets/partials/features/_card.scss [hunk 5]
```diff
@@ -1,110 +1,110 @@
-.cards {
-	margin: 1em 0 2em;
-	float: left;
-	width: 100%;
+// .cards {
```
Comment (mxsnyder) [inline stylesheets/partials/features/_card.scss:1]: Again - commented out on purpose?

**Review by mxsnyder — DISMISSED** [review]
Nothing critical here, just some spacing and questions about commented out code, so I'll approve.

File: gulpfile.js (hunk 1 above)
Comment (martypowell) [reply gulpfile.js]: yes, i plan on using all of your tasks...but i'm not ready to add any angular in yet.

File: stylesheets/partials/features/_card.scss (hunk 5 above)
Comment (martypowell) [reply stylesheets/partials/features/_card.scss:1]: yes, there is a card component that is different from bcpl, but i plan on using

File: js/mobile-nav.js (hunk 2 above)
Comment (martypowell) [reply js/mobile-nav.js]: fixed

File: js/mobile-nav.js (hunk 3 above)
Comment (martypowell) [reply js/mobile-nav.js]: good call, done

**Review by mxsnyder — APPROVED** [review]

**Pull request merged**